server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/SpanServiceElasticsearch.java

```diff
@@ -246,7 +246,7 @@ public Trace getTraceFragment(String tenantId, String id) {
     @Override
     public Trace getTrace(String tenantId, String id) {
         Span span = getSpan(tenantId, id);
-        if (span != null && span.serverSpan()) {
+        if (span == null || (span != null && span.serverSpan())) {
```

Comment:
`(span == null || (span != null && span.serverSpan()))` is the same as `(span == null || span.serverSpan())`